falcon/request.py

```diff
@@ -514,6 +522,58 @@ def cookies(self):
 
         return self._cookies.copy()
 
+    def param(self):
```

Comment:
I'd be in favor of that -- adding the mapping interface to ParamProxy should make it a relatively small change, and definitely acceptable in a major version bump IMHO.